userspace/libsinsp/sinsp_ppm_sc.cpp

```diff
@@ -238,6 +251,41 @@ std::unordered_set<uint32_t> sinsp::enforce_sys_ppm_sc_set(std::unordered_set<ui
 	return ppm_sc_set;
 }
 
+std::unordered_set<uint32_t> sinsp::enforce_events(std::unordered_set<uint32_t> ppm_event_info_of_interest)
```

Comment:
I see now; perhaps we can call this one (following other existing method) `enforce_sinsp_state_ppme`?